docs/learn/digital-assets/migrate-erc20-to-lsp7.md

```diff
@@ -0,0 +1,194 @@
+| _No equivalent_                                          | `revokeOperator(address spender, bytes memory data)`                                |
+| _No equivalent_                                          | `increaseAllowance(address spender, uint256 addedAmount, bytes memory data)`        |
+| _No equivalent_                                          | `decreaseAllowance(address spender, uint256 subtractedAmount, bytes memory data)`   |
+| `transfer(address to, uint256 amount)`                   | `transfer(address from, address to, uint256 amount, bool force, bytes memory data)` |
+| `transferFrom(address from, address to, uint256 amount)` | `transfer(address from, address to, uint256 amount, bool force, bytes memory data)` |
+| _No equivalent_                                          | `batchCalls(bytes[] memory data)`                                                   |
+
+To check function definitions and explanations of behavior and each parameter, check [API Reference](../../contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.md) section
+
+## dApps and Indexers
+
+To track the events of a LSP7DigitalAsset contract, different event definitions should be used for listening. This is a table comparing event definitions:
+
+| ERC20 Event                                                               | LSP7 Event                                                                                                                                   |
+| ------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------- |
+| `Transfer(address indexed from, address indexed to, uint256 value)`       | `Transfer(address indexed operator, address indexed from, address indexed to, uint256 amount, bool force, bytes data)`                       |
+| `Approval(address indexed owner, address indexed spender, uint256 value)` | `OperatorAuthorizationChanged(address indexed operator, address indexed tokenOwner, uint256 indexed amount, bytes operatorNotificationData)` |
+| _No equivalent_                                                           | `OperatorRevoked(address indexed operator, address indexed tokenOwner, bool indexed notified, bytes operatorNotificationData)`               |
+
+To check event definitions and explanations of behavior and each parameter, check [API Reference](../../contracts/contracts/LSP7DigitalAsset/LSP7DigitalAsset.md) section
+
+## Metadata Management
+
+### Basic Token Information
+
+In ERC20, the name and symbol of a token can be retrieved by calling their own function:
+
+```javascript
+// ERC20
+const name = await token.name();
+const symbol = await token.symbol();
+```
+
+In LSP7, the name and symbol of the token will be retrieved with getData function, since the LSP7 uses a generic metadata key value store:
```

Comment:
Suggest adding link to `getData` function ABI.

```suggestion
In LSP7, the token name and symbol can be retrieved with getData function, since LSP7 uses a generic metadata key value store:
```